bedrock/styleguide/templates/styleguide/communications/copy-rules.html

```diff
@@ -28,175 +28,175 @@
-    <dt>Firefox Nightly</dt>
+    <dt id="firefox-nightly">Firefox Nightly</dt>
     <dd>Firefox Nightly on first mention, Nightly on subsequent mentions, always capitalized</dd>
-    <dt>Firefox OS</dt>
+    <dt id="firefox-os">Firefox OS</dt>
     <dd>the official name of the project formerly known as Boot to Gecko (or B2G); always use the complete name; do not use “the Firefox OS” unless it is being treated as an adjective (e.g. the Firefox OS experience, the Firefox OS home screen, etc.)</dd>
-    <dt>Firefox Themes</dt>
+    <dt id="firefox-themes">Firefox Themes</dt>
     <dd>themes is capitalized when it follows Firefox, lowercase when on its own; it can be singular or plural as needed and is always localized; formerly “Firefox Personas”</dd>
-    <dt>free</dt>
+    <dt id="free">free</dt>
     <dd>this one is tricky as it means both “without cost” and “without restrictions” in English, though there are separate words for these concepts in many other languages; please be very clear when using or avoid</dd>
-    <dt>fullscreen</dt>
+    <dt id="fullscreen">fullscreen</dt>
     <dd>one word, lowercase, when referring to the view mode of the browser; use “entire screen,” “whole screen” or similar formulation in other cases</dd>
-    <dt>FxOS</dt>
+    <dt id="fxos">FxOS</dt>
     <dd>for internal usage, this is the official abbreviation of “Firefox OS,” but it should not be used in any external communications</dd>
-    <dt>gear</dt>
+    <dt id="gear">gear</dt>
     <dd>used to refer to any branded merchandise, be it Mozilla, Firefox or any of our other brands; please use instead of “swag” or “merchandise”</dd>
-    <dt>http://</dt>
+    <dt id="http://">http://</dt>
     <dd>this should never appear before a URL in communications unless there is a very good reason (though we can’t think of one off the tops of our heads)</dd>
-    <dt>i18n</dt>
+    <dt id="i18n">i18n</dt>
     <dd>our abbreviation for internationalization (18 letters between the I and N), reserved for internal use or where space is a consideration, otherwise please avoid or explain on first use</dd>
-    <dt>Internet</dt>
+    <dt id="internet">Internet</dt>
     <dd>always capitalized</dd>
-    <dt>Join Mozilla</dt>
+    <dt id="join-mozilla">Join Mozilla</dt>
     <dd>refers specifically to the fundraising campaign and should not be used in other contexts; both words are capitalized and “Join” should never become “Joins,” “Joining” or any other form of the word</dd>
-    <dt>l10n</dt>
+    <dt id="l10n">l10n</dt>
     <dd>our abbreviation for localization (10 letters between the L and N), reserved for internal use or where space is a consideration, otherwise please avoid or explain on first use</dd>
-    <dt>left-click</dt>
+    <dt id="left-click">left-click</dt>
     <dd>hyphenated in all forms and uses</dd>
-    <dt>merch/merchandise</dt>
+    <dt id="merch/merchandise">merch/merchandise</dt>
```

Comment:
Actually, we could probably just use `merchandise` here instead to keep it simple.